llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -1282,8 +1283,14 @@ class SegmentBuilder {
       // value for that area.
       // We add counts of the regions of the same kind as the active region
       // to handle the both situations.
-      if (I->Kind == Active->Kind)
-        Active->ExecutionCount += I->ExecutionCount;
+      if (I->Kind == Active->Kind) {
+        assert(I->HasSingleByteCoverage == Active->HasSingleByteCoverage &&
+               "Regions are generated in different coverage modes");
```

Comment:
This is just a minor nit that I just noticed and could be addressed in some follow up change, but in all other `assert` cases the message is not capitalized so shouldn't capitalize here either for consistency.